SmallLibrary/js/main.js

```diff
@@ -0,0 +1,122 @@
+var Main = function () {
```

Comment:
It will be very useful to have some separated modules here.
For example:
1) module which describes all test data that we need
2) module for populating of test data
3) the main module which uses all other modules:
(function(){	
	var testData = new TestDataForMiniJsLibrary();
	var tests = new TestsForMiniJsLibrary(testData); 
	
	var populateTestData = function() {
		var testData = new TestDataForMiniJsLibrary();
		testData.array = $('.array').html();
	};
	
	$('.button').on('click', populateTestData);
	
	$('.findLastElement').on('click', tests.findLastElement);	
})();